### PR TITLE
Update gurumdds rosdep key for jammy

### PR DESCRIPTION
**Pull request by YoungJin-gurum**
Please add the following dependency to the rosdep database.

## Package name:

gurumdds

## Package Upstream Source:

[GurumDDS](https://gurum.cc/gurumdds) is an rmw provider distributed in the ROS bootstrap repository.

## Purpose of using this:

This package is used by [rmw_gurumdds](https://github.com/ros2/rmw_gurumdds). 

Distro packaging links:

## Links to Distribution Packages

- Debian: https://packages.debian.org/
  - not available
- Ubuntu: https://packages.ubuntu.com/
   - http://repos.ros.org/repos/ros_bootstrap/pool/main/g/gurumdds-2.6/
   - http://repos.ros.org/repos/ros_bootstrap/pool/main/g/gurumdds-2.7/
   - http://repos.ros.org/repos/ros_bootstrap/pool/main/g/gurumdds-2.8/
- Fedora: https://src.fedoraproject.org/browse/projects/
  - NOT AVAILABLE
- Arch: https://www.archlinux.org/packages/
  - NOT AVAILABLE
- Gentoo: https://packages.gentoo.org/
  - NOT AVAILABLE
- macOS: https://formulae.brew.sh/
  - NOT AVAILABLE
- Alpine: https://pkgs.alpinelinux.org/packages
  - NOT AVAILABLE
- NixOS/nixpkgs: https://search.nixos.org/packages
  - NOT AVAILABLE

### CODE REVIEW

**Comment by YoungJin-gurum** [comment]
I have asked @nuclearsandwich  to pull the latest version of the Gurumdds package.

**Comment by nuclearsandwich** [comment]
Closing and re-opening to trigger actions.

File: rosdep/base.yaml [hunk 1]
```diff
@@ -1711,6 +1711,7 @@ gurumdds-2.8:
     bionic: [gurumdds-2.8]
     focal: [gurumdds-2.8]
     xenial: [gurumdds-2.8]
+    jammy: [gurumdds-2.8]
```
Comment (nuclearsandwich) [inline rosdep/base.yaml:1714]:
```suggestion
    bionic: [gurumdds-2.8]
    focal: [gurumdds-2.8]
    jammy: [gurumdds-2.8]
    xenial: [gurumdds-2.8]
```

Fix lexicographical sorting.

Comment (YoungJin-gurum) [reply rosdep/base.yaml]: @nuclearsandwich I've done.

**Review by tfoote — APPROVED** [review]
CI is passing now. @nuclearsandwich I'll leave it to you to verify and merge.

**Review by nuclearsandwich — APPROVED** [review]

**Pull request merged**